black_it/utils/time_series.py

```diff
@@ -81,3 +84,41 @@ def get_mom_ts_1d(time_series: NDArray[np.float64]) -> NDArray[np.float64]:
     np.nan_to_num(avg_vec_mom, False)
 
     return avg_vec_mom
+
+
+def hp_filter(
+    time_series: NDArray[np.float64], lamb: float = 1600
+) -> Tuple[NDArray[np.float64], NDArray[np.float64]]:
+    """
+    Apply the HP filter to a time series.
```

Comment:
Might be a good idea to put a bibliographic reference here?

Reply:
Yes, thanks